src/workerd/api/crypto/crypto.h

```diff
@@ -10,8 +10,8 @@
 #include <workerd/jsg/buffersource.h>
 #include <workerd/jsg/jsg.h>
 #include <workerd/jsg/buffersource.h>
-#include <openssl/err.h>
-#include "../streams.h"
+#include "openssl/base.h"  // for EVP_MD_CTX, X509
```

Comment:
nit:
```suggestion
#include <openssl/base.h>  // for EVP_MD_CTX, X509
```

Reply:
I blame Google's style guide for these 😄 – they eschew the use of angled brackets for some reason, and since IWYU was developed with Google3 in mind that's what it is producing.

Reply:
It's actually our fault, I tried to fix it but I don't understand bazel well enough.
IWYU will include with "" if file is in `-I` include path and with <> if file is in `-isystem` include path.
Ideally I'd like our bazel configuration to include all these libraries as ONLY `-isystem` instead of the current behavior which is both